docs/OSS.md

```diff
@@ -0,0 +1,7 @@
+# OSS
+
+If you want to use OSS storage for the sccache cache, you need to set the `SCCACHE_OSS_BUCKET` environment variable to the name of the OSS bucket to use.
+
+You **must** configure the region using the `SCCACHE_ENDPOINT` environment variable.
```

Comment:
Are we sharing the same `SCCACHE_ENDPOINT` environment? Seems we are using `SCCACHE_OSS_ENDPOINT`